ocaml/xapi/xapi_xenops.ml

```diff
@@ -2438,27 +2432,19 @@ let update_vm ~__context id =
 let update_vbd ~__context (id : string * string) =
   try
     if Events_from_xenopsd.are_suppressed (fst id) then
-      debug "xenopsd event: ignoring event for VM (VM %s migrating away)"
+      debug "xenopsd event: ignoring event for VBD (VM %s migrating away)"
```

Comment:
I presume this was originally a copy-paste error, VM instead of VBD? The same with s/VM/VUSB migrating away later